### PR TITLE
zh: add content_zh/about/notes/1.1.1

### PR DESCRIPTION
**Pull request by SataQiu**

### CODE REVIEW

**Comment by fleeto** [comment]
switch this PR to release-1.1 pls.

**Comment by fleeto** [comment]
`[trans-bot:N/A]`

Thank you @SataQiu, there are more than 1 files in your PR, will not be reported to the task issues.

File: content_zh/about/notes/1.1.1/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- 配置 Prometheus 监控 Citadel ([Issue 12175](https://github.com/istio/istio/pull/12175))
+- 改进 [`istioctl experimental verify-install`](/zh/docs/reference/commands/istioctl/) 命令的输出 ([Issue 12174](https://github.com/istio/istio/pull/12174))
+- 降低 SPIFFE URI 丢失服务账户的消息日志级别 ([Issue 12108](https://github.com/istio/istio/issues/12108))
+- 修复 SDS 功能在 Unix 系统上损坏的路径 ([Issue 12688](https://github.com/istio/istio/pull/12688))
+- 修复 Envoy 追踪中如果父 span 使用空字符串传播子 span 的创建被阻止的问题 ([Envoy Issue 6263](https://github.com/envoyproxy/envoy/pull/6263))
+- 为网关端口添加命名空间范围。这修复了两个问题：
+    - `IngressGateway` 只关注第一个端口 443 网关定义 ([Issue 11509](https://github.com/istio/istio/issues/11509))
+    - Istio `IngressGateway` 因为具有相同端口的不同网关（SDS）发生路由失败([Issue 12500](https://github.com/istio/istio/issues/12500))
+- 地点加权负载均衡的五个 Bug 修复：
+    - 修复导致任意地点出现空端点的 Bug ([Issue 12610](https://github.com/istio/istio/issues/12610))
```
Comment (fleeto) [inline content_zh/about/notes/1.1.1/index.md]: 修复了为每个区域生成空端点的问题。

**Comment by SataQiu** [comment]
@fleeto 解决了冲突问题，还请再 review 下吧！

**Pull request closed without merging**